setup.py

```diff
@@ -19,7 +19,7 @@
         'shapely>=1.6b3'
     ],
     include_package_data=True,
-    data_files=[('jars', ['geopyspark/jars/geotrellis-backend-assembly-0.1.0.jar'])],
+    # data_files=[('jars', ['geopyspark/jars/geotrellis-backend-assembly-0.1.0.jar'])],
```

Comment:
Ok, interesting. I think that works out. I just don't want to lose the ability to run GeoPySpark easily in local mode when getting the EMR options.